witchcraft/internal/wdebug/handlers.go

```diff
@@ -76,7 +78,7 @@ func (h handlerGoroutinesV1) SafeLoggable() bool {
 }
 
 func (h handlerGoroutinesV1) Extension() string {
-	return "txt"
+	return "prof"
```

Comment:
Is this useful somehow on the old format or can we keep it as-is?

Reply:
oops meant to revert this

Reply:
fixed